plugins/modules/docker_container.py

```diff
@@ -38,7 +38,7 @@
     type: int
   capabilities:
     description:
-      - List of capabilities to add to the container.
+      - List of capabilities to add to the container (equivalent to docker cap_add).
```

Comment:
What exactly do you mean with `docker cap_add`? That's apparently not a CLI command (also the CLI option is called `--cap-add`).

Reply:
Ah yes looks like I'm conflating the docker-cli & docker-compose syntax. It's `cap_add` in compose for example as used here: https://hub.docker.com/r/pihole/pihole.

Would this be preferable: 

(equivalent to `docker --cap-add`, or `docker-compose` option `cap_add`).

Reply:
I would mention `docker run --cap-add`, something like this:
```suggestion
      - List of capabilities to add to the container.
      - This is equivalent to C(docker run --cap-add).
```

Reply:
This is fine except that the original reason I discovered that this parameter was named differently from docker cli or compose was that when I searched the docs for the literal string `cap_add` whilst trying to translate a docker-compose file to an ansible playbook I didn't get a hit. I suspect others trying to switch to ansible might take the same path so it seems like a good idea to me to have the names of both the cli and compose equivalents in the docs. I'm new here so I don't know if the community has existing conventions around this and having both might get a bit bloated but I'd make it something that has the literal strings from both cli and compose e.g. 

`- This is equivalent to C(docker run --cap-add), or the C(docker-compose) option C(cap_add).`

Happy to go with your suggestion if you think this is still to vague and making it more specific about the compose equivalent would be to verbose.

Reply:
How about: `- This is equivalent to C(docker run --cap-add), or the docker-compose option C(cap_add).`?

Reply:
You still need to remove the part from this line :)
```suggestion
      - List of capabilities to add to the container.
```